pages/docs/pydicom/viewing_images.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Visualizing DICOM images using Pydicom
+sidebar: pydicom
+permalink: pydicom-visualization
+folder: docs
+---
+
+
+## Introduction
+
+pydicom is mainly concerned with getting at the DICOM data elements in files,
+but it is often desirable to view pixel data as an image.
+There are several options:
+
+  * Use any of the many `DICOM viewer
+    <http://www.dclunie.com/medical-image-faq/html/part8.html#DICOMFileConvertorsAndViewers>`_
+    programs available
+  * use pydicom with `matplotlib <http://matplotlib.sourceforge.net/>`_
+  * use pydicom with python's stdlib `Tkinter <https://docs.python.org/3.4/library/tkinter.html>`_ module.
+  * use pydicom with the `Python Imaging Library (PIL)
+    <http://www.pythonware.com/products/pil/>`_
+  * use pydicom with `wxPython <http://www.wxpython.org/>`_
+
+## Using pydicom with matplotlib
+
+matplotlib is available at http://matplotlib.sourceforge.net/. It
+can take 2-d image information from ``Dataset.pixel_array`` and display it.
+Here is an example::
+
+```python
+>>> import pydicom
+>>> import pylab
```

Comment:
agreed! Do you want to give a go at using markdown? Im not  sure if rst syntax will work for jekyll. Did you get it set up ok? I can add a page  for how to contribute to the docs with documentation for setting up and using Jekyll